arviz/tests/base_tests/test_plot_utils.py

```diff
@@ -23,6 +23,10 @@
 from ..helpers import running_on_ci
 
 
+# Check if Bokeh is installed
+bokeh_installed = importlib.util.find_spec("bokeh") is not None  # pylint: disable=invalid-name
```

Comment:
I just moved this to the top (it used to be in the middle of the file) and then made all the `skipif` decorators in this file use the same logical expression as the one on line 310. I chose the expression from that line since it seemed the most obvious and explicit to me, but I'm happy to switch to a different one if the maintainers have a different preference.